src/titiler/core/pyproject.toml

```diff
@@ -46,7 +46,7 @@ test = [
     "pytest",
     "pytest-cov",
     "pytest-asyncio",
-    "httpx",
+    "httpx==0.24.1",
```

Comment:
`0.25.0` was causing 2 test failures. What're your thoughts on leaving this pinned, and seeing if it gets resolved upstream?

Reply:
sounds good, but make a note about why it's pinned